storj/downloader.py

```diff
@@ -36,15 +38,16 @@ def set_file_metadata(self, bucket_id, file_id):
             file_metadata = self.client.file_metadata(bucket_id, file_id)
             # Get file name
             self.filename_from_bridge = str(file_metadata.filename)
-            print "Filename from bridge: " + self.filename_from_bridge
+            self.__logger.debug("Filename from bridge: " +
```

Comment:
in this case you can do:

```
self.__logger.debug("Filename from bridge: %s", self.filename_from_bridge)
```

the format and values will then be replaced by the `logging` library if `debug` is enabled.

this differs from the cases where you just want to build a string.

for example:
```
msg = 'Filename from bridge: %s' % self.filename_from_bridge
```

_remember, single quotes_ :)